src/line-chart/line-chart.js

```diff
@@ -3,14 +3,22 @@ import Chart from '../chart/chart'
 
 class LineChart extends Chart {
     createPaths({ data, x, y }) {
-        const { curve } = this.props
-
-        const line = shape
-            .line()
-            .x((d) => x(d.x))
-            .y((d) => y(d.y))
-            .defined((item) => typeof item.y === 'number')
-            .curve(curve)(data)
+        const { curve, continuousLine } = this.props
+        let line
+        if (continuousLine) {
+            line = shape
+                .line()
+                .x((d) => x(d.x))
+                .y((d) => y(d.y))
+                .curve(curve)(data.filter((item) => item.y))
+        } else {
+            line = shape
+                .line()
+                .x((d) => x(d.x))
+                .y((d) => y(d.y))
+                .defined((item) => typeof item.y === 'number')
+                .curve(curve)(data)
+        }
```

Comment:
Wouldn't this work? 
Basically it exposes the `defined` function to user land, allowing them to implement it anyway they see fit 
```suggestion
        const { 
          defined = (item) => typeof item.y === 'number' ,
          continuousLine
          } = this.props
        const shape =
                .line()
                .x((d) => x(d.x))
                .y((d) => y(d.y))
                .defined(defined)
                .curve(curve)(data)
```

Reply:
Hi, can you explain how can I use `defined` function for this situation?

Reply:
Hi, my suggestion clearly states that `defined` is exposed as a prop. Meaning that we can have a default value (the one it is currently)  and that the user can override it how they see fit

Reply:
I tried to implement the `defined` function for this specific use case but I failed. Do you have any idea?
That would help me to solve the main problem.

Reply:
If you could provide a PR with your attempt and clearly stating what isn't working I might be able to help you 👍

Reply:
Can you help me with this?
Actually I think it's impossible to handle this use case without filtering data.

Reply:
I unfortunately don't have any time to assist outside of reviewing the PR. The user controls the data so I don't see why the user won't be able to filter the data? What exactly are you referring to in your example?

Reply:
@JesperLekland For example, we want to show the week, and the data is only available for Monday to Thursday. Line Chart will not be able to connect Monday and Thursday.

Reply:
it's not only ` .defined(defined)` also the data being fed to the chain is different.